nf_core/lint/multiqc_config.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+import os
+import yaml
+
+
+def multiqc_config(self):
+    """Make sure basic multiQC plugins are installed and plots are exported
+    Basic template:
+    report_comment: >
+        This report has been generated by the <a href="https://github.com/nf-core/quantms" target="_blank">nf-core/quantms</a>
+        analysis pipeline. For information about how to interpret these results, please see the
+        <a href="https://nf-co.re/quantms" target="_blank">documentation</a>.
+    report_section_order:
+        software_versions:
+            order: -1000
+        nf-core-quantms-summary:
+            order: -1001
+
+    export_plots: true
+    """
+    passed = []
+    failed = []
+
+    fn = os.path.join(self.wf_path, "assets", "multiqc_config.yaml")
+
+    # Return a failed status if we can't find the file
+    if not os.path.isfile(fn):
+        return {"failed": ["'assets/multiqc_config.yaml' not found"]}
```

Comment:
The test is only about the formatting. Existence is already tested for elsewhere:

https://github.com/nf-core/tools/blob/b1ce47148a05965e0604534cece3359d408af357/nf_core/lint/files_exist.py#L144

It's annoying to test it in two different places in case people want to configure the linting to allow it to not be there (eg. if the pipeline doesn't use MultiQC).

```suggestion
        return {"ignored": ["'assets/multiqc_config.yaml' not found"]}
```

Example elsewhere: https://github.com/nf-core/tools/blob/b1ce47148a05965e0604534cece3359d408af357/nf_core/lint/actions_ci.py#L76-L78

Reply:
Ah ok. I was confused that the other ones would just ignore.

Reply:
Yeah, if spot any that don't ignore feel free to switch them (I think I saw a warning somewhere). Could probably add better comments to explain too 🤔